pttrack/templates/pttrack/workup_detail.html

```diff
@@ -22,7 +22,9 @@
     <div class="row text-center">
         <h4>Actions</h4>
         <div class="btn-group">
+            {% if workup.signed and request.session.clintype_pk == "Attending" or not workup.signed %}
```

Comment:
I am wondering if we could come up with a way of doing this that doesn't require a string comparison? Could we do like `request.session.clintype.signs_charts` or something? I don't really remember the interface...